server/server.js

```diff
@@ -100,4 +98,19 @@ app.get('/tasks', (req, res) =>
         })
 );
 
+app.post('/taskUpdated', (req, res) => {
+    Task.findById(req.body._id, (err, task) => {
```

Comment:
Use promise version of `.findOneAndUpdate` instead - [docs](https://mongoosejs.com/docs/api.html#model_Model.findOneAndUpdate)